samples/command/provider/src/provider_impl.rs

```diff
@@ -28,29 +28,9 @@ impl Provider for ProviderImpl {
         &self,
         request: Request<SubscribeRequest>,
     ) -> Result<Response<SubscribeResponse>, Status> {
-        let request_inner = request.into_inner();
-        let entity_id: String = request_inner.entity_id.clone();
-        let consumer_uri: String = request_inner.consumer_uri;
-
-        info!(
-            "Received a subscribe request from consumer URI {} for entity id {}",
-            &consumer_uri, &entity_id
-        );
-
-        let mut lock: MutexGuard<HashMap<String, HashSet<String>>> =
-            self.subscription_map.lock().unwrap();
-        let mut uris = match lock.get(&entity_id) {
-            Some(get_value) => get_value.clone(),
-            None => HashSet::new(),
-        };
-        uris.insert(consumer_uri);
-        lock.insert(entity_id, uris);
-
-        info!("Completed subscription.");
+        warn!("Got a subscribe request: {:?}", request);
 
-        let response = SubscribeResponse {};
-
-        Ok(Response::new(response))
+        Err(Status::unimplemented("subscribe has not been implemented"))
```

Comment:
why was the implementation removed?

Reply:
The command sample does not use pub-sub, instead it just uses invoke-respond.